threadit-web/src/pages/PostSpool/PostSpool.tsx

```diff
@@ -40,8 +44,13 @@ export default function PostSpool() {
                                     <Input disabled={lockInputs} size='md' value={title} onChange={(e) => setTitle(e.target.value)} />
                                 </FormControl>
                                 <FormControl>
-                                    <FormLabel>Interests</FormLabel>
-                                    <Input disabled={lockInputs} size='md' value={interests} onChange={(e) => setInterests(e.target.value)} />
+                                    {/*<FormLabel>Interests</FormLabel>*/}
+                                    {/*<Input disabled={lockInputs} size='md' value={interests} onChange={(e) => setInterests(e.target.value)} />*/}
+
```

Comment:
Can this commented out code be removed?

Reply:
it has been already, I think you're looking at a previous commit and not the most recent updates